server/views/register.ejs

```diff
@@ -1,30 +1,28 @@
 <%var title = 'Login'%>
 
+<link rel="stylesheet" type="text/css" href="./css/register.css">
 
-<h1>Register</h1>
-<form action="/api/v1/users/register" method="POST">
-    <div>
-        <label for="name">Name</label>
-        <input type="name" name="name" placeholder="Enter Name" value="<%= typeof name != 'undefined' ? name : '' %>" />
-
-    </div>
-    <div>
-        <label for="email">Email</label>
-        <input type="email" name="email" placeholder="Enter Email"
-            value="<%= typeof email != 'undefined' ? email : '' %>" />
-    </div>
-    <div>
-        <label for="password">Password</label>
-        <input type="password" name="password" placeholder="Create Password"
-            value="<%= typeof password != 'undefined' ? password : '' %>" />
-    </div>
-    <div>
-        <label for="password2">Confirm Password</label>
-        <input type="password" name="password2" placeholder="Confirm Password"
-            value="<%= typeof password2 != 'undefined' ? password2 : '' %>" />
-    </div>
-    <button type="submit" class="btn btn-primary btn-block">
-        Register
-    </button>
-</form>
-<p>Have An Account? <a href="/login">Login</a></p>
\ No newline at end of file
+<div class="container">
+    <h1>Register</h1>
+    <form action="/api/v1/users/register" method="POST">
+        <div>
+            <input type="name" name="name" class="text-input" placeholder="Enter Name" value="<%= typeof name != 'undefined' ? name : '' %>" />
+        </div>
+        <div>
+            <input type="email" name="email" class="text-input" placeholder="Enter Email"
+                value="<%= typeof email != 'undefined' ? email : '' %>" />
```

Comment:
Consider making a class for every row and using something like `margin-top: 5px;`